.github/workflows/email.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         id: changed-files
         uses: tj-actions/changed-files@v32
         with:
-          separator: ","                 
+          separator: "\n"                 
```

Comment:
To address "Can you split these with newlines instead of commas?"
Added newline as separator.